tests/pyrog/migrations/test_0003_unique_source_name.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+
+@pytest.mark.django_db
+@pytest.mark.migration_test
+@pytest.mark.migration(app_label="pyrog", migration_name="0002_integrate_users")
+def test_migrate(migrator, state):
+    Template = state.apps.get_model("pyrog", "Template")
+    Source = state.apps.get_model("pyrog", "Source")
+
+    t1 = Template.objects.create(name="template_1")
+    t2 = Template.objects.create(name="template_2")
+
+    s1 = Source.objects.create(template=t1, name="my_source")
+    s2 = Source.objects.create(template=t2, name="my_source")
+
+    try:
+        migrator.apply_tested_migration(("pyrog", "0003_unique_source_name"))
+    except Exception:
+        pytest.fail()
```

Comment:
why did you have to do this ? the test will fail if this throws an exception
also, it looks like you swallowed the error (the exception isn't re-raised)

Reply:
It was there before but I agree that it's a bit weird, I'll remove that 🤷‍♂️